nox/sessions.py

```diff
@@ -632,7 +632,12 @@ def install(self, *args: str, **kwargs: Any) -> None:
         if "silent" not in kwargs:
             kwargs["silent"] = True
 
-        self._run("python", "-m", "pip", "install", *args, external="error", **kwargs)
+        if isinstance(venv, VirtualEnv) and venv.venv_backend == "uv":
+            self._run("uv", "pip", "install", *args, external="error", **kwargs)
```

Comment:
Should this default to `--upgrade`? `uv` does not seem to install any new dependencies, without `--upgrade`. This is only a problem if you use `reuse_existing_virtualenvs=True`.

Since `uv` is really fast on subsequent runs, it probably does not matter though (or, can be passed explicitly if you really want to).

Reply:
BTW, I believe the venv is "active" here, if not, we need to set `VIRTUAL_ENV`. `uv pip` doesn't have a `--python=.venv` option at the moment.

Reply:
`VIRTUAL_ENV` is always in the env, I believe, so this is fine